src/transformers/trainer_pt_utils.py

```diff
@@ -154,6 +154,7 @@ def distributed_concat(tensor: "torch.Tensor", num_total_examples: Optional[int]
             return type(tensor)(distributed_concat(t, num_total_examples) for t in tensor)
         output_tensors = [tensor.clone() for _ in range(dist.get_world_size())]
         dist.all_gather(output_tensors, tensor)
+        output_tensors = [t if len(t.shape) > 0 else t.reshape((-1,)) for t in output_tensors]
```

Comment:
```suggestion
        output_tensors = [t if len(t.shape) > 0 else t[None] for t in output_tensors]
```
nit: does the same thing but I thing it's more readable like this.